<launch>
    <node name="yolov5_object_detector" pkg="ros_yolov5" type="ros_yolov5_service.py" output="screen">
        
        <param name="yolo_path" value="$(find ros_yolov5)/yolov5" />
        <param name="weights_path" value="$(find ros_yolov5)/weights/yolov5s.pt" />
        <param name="detection_image_topic" value="/object_detection/image" />
        <param name="detection_counter_topic" value="/object_detection/counter" />
        <param name="detection_names_topic" value="/object_detection/names" />
        <param name="detection_objects_topic" value="/object_detection/objects" />
        <param name="source_topic" value="/camera/rgb/image_raw" />
        <param name="source_queue" value="1" />
        <param name="as_a_service" value="true" />
        <param name="service_name" value="object_detection/detect_object" />
    </node>
</launch>
